package.json

```diff
@@ -75,6 +75,7 @@
   "scripts": {
     "build": "gulp build",
     "lint": "gulp lint sass-lint",
+    "package": "cd addon && npm run package",
```

Comment:
I'm guessing this would only work if you did a `$ cd addon && npm install` first though, right?
I think I'd see an error if I just did this from a fresh `git clone`:
```sh
npm install && npm run package
```

Of course, I should probably verify that theory, because there is nothing worse than some drive-by QA person starting two sentences with "I'm guessing" and "I think" like the lazy jerk they are... #thisguy